### DISCUSSION TITLE
🎉 Welcome to Skyvern Community ⭐

### DISCUSSION THREAD

**Discussion by ykeremy** [Skyvern-AI/skyvern#75] [Announcements]
Welcome to the Skyvern Community!

As we grow and evolve, it's essential to keep our communication channels organized. Here’s how we can best use each platform:

1. **Discord**: This is the place for specific support requests and real-time interaction. Share your experiences, what you've tried, your successes, and keep updated with Skyvern milestones. It's a vibrant space for support and sharing! [Join our Discord.](https://discord.com/invite/fG2XXEuQX3)

2. **GitHub Discussions**: Consider this our knowledge hub. Engage in conversations like those on our [HN post](https://news.ycombinator.com/item?id=39706004), ask questions, and explore 'FAQs'. It’s perfect for in-depth discussions and learning from each other.

3. **GitHub Issues**: Use this for reporting bugs or suggesting new features. It’s our primary tool for tracking improvements and keeping Skyvern bug-free.

Let's collaborate effectively across these platforms to make Skyvern even more powerful! 🚀

**Comment by Welding-Torch** [comment]
Nice. I'm hoping for an easier way to get started with Skyvern.

**Comment by neil-s** [comment]
This is awesome! What's the best place to report / submit logs from individual failed runs of the hosted version? It feels like it doesn't quite rise to the level of a Github issue, but it might be helpful for your prompt engineering / testing.
